### DISCUSSION TITLE
Default workflows?

### DISCUSSION THREAD

**Discussion by Hujikuio** [jordenyt/stable_diffusion_sketch#24] [Ideas]
It would be nice if you could provide some simple workflows that would go with the example/default .jsons in the app for the txt2img, img2img, inpaint, etc. features on the gateway github page. I think it would be a helpful addition, but I'm not complaining at all, I love that you made this app and I'll figure it out eventually. I've been learning comfyui and was very happy to see that this app is heading that direction as well :)

**Comment by jordenyt** [comment]
There are some examples workflow for SDXL in the `\workflows` folder of the project ComfyUIGW, which you can treat it as sample for txt2img, img2img and inpainting.

Yes, I have got my own workflows for Flux, SUPIR, SD3.5, icLight, IDM-VTON........ If you want it, I may share it in the discussions.

**Reply by Hujikuio** [reply]
> There are some examples workflow for SDXL in the `\workflows` folder of the project ComfyUIGW, which you can treat it as sample for txt2img, img2img and inpainting.
> 
> Yes, I have got my own workflows for Flux, SUPIR, SD3.5, icLight, IDM-VTON........ If you want it, I may share it in the discussions.

That would be awesome if you could. I have a few of my own workflows I've made, but some more fleshed out references on how to convert them to play nicely with the app would be incredibly helpful, especially for inpainting workflows.

**Reply by Hujikuio** [reply]
I guess I'm also assuming the Node ID's from the API conversion aren't arbitrarily assigned from client to client after exporting. If they are hard coded or easily remapped, then having the non converted template workflow for Comfy and the associated template API workflow for the Gateway would be a really amazing template/reference. Sorry if I'm misunderstanding things or overcomplicating it, I finally stopped putting off ComfyUI and dove in head first, and am still a bit overwhelmed. Thank you for your app though, both A1111 and Comfy versions are incredible, and yours easily has the best inpainting support for Android.

**Comment by jordenyt** [comment]
@Hujikuio I have put the FLUX workflow in a new discussion topics.  Please check if you can follow.
